hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/commandhandler/DeleteBlocksCommandHandler.java

```diff
@@ -486,6 +489,7 @@ private void logDeleteTransaction(long containerId,
     }
 
     if (delTX.getTxID() <= containerData.getDeleteTransactionId()) {
+      blockDeleteMetrics.incrFailureCount();
```

Comment:
I guess this should increment the new metric.

```suggestion
      blockDeleteMetrics.incOutOfOrderDeleteBlockTransactionCount();
```

Reply:
@adoroszlai Updated...